pip_audit/_format/cyclonedx.py

```diff
@@ -75,6 +74,10 @@ def format(self, result: Dict[service.Dependency, List[service.VulnerabilityResu
         parser = _PipAuditResultParser(result)
         bom = Bom.from_parser(parser)
 
-        formatter = output.get_instance(bom=bom, output_format=self._inner_format.value)
+        formatter = output.get_instance(
+            bom=bom,
+            output_format=self._inner_format.value,
+            schema_version=output.SchemaVersion.V1_4,
```

Comment:
N.B.: This explicitly pins us to CycloneDX's 1.4 schema now that it's supported, which fixes #151.